include/avk/commands.hpp

```diff
@@ -1107,31 +1107,56 @@ namespace avk
 #endif
 	}
 
+	/** Helper struct to specify a semaphore and a signal value for it.
+	 *  This is used for timeline semaphores.
+	 */
+	struct semaphore_value_info
+	{
+		avk::resource_argument<avk::semaphore_t> mSemaphore;
+		uint64_t mValue;
+	};
+	inline semaphore_value_info operator,(avk::resource_argument<avk::semaphore_t> aSemaphore, uint64_t aValue) {

```

Comment:
Please add exactly one newline between each function/method! 
Also make sure to add comments!